package.json

```diff
@@ -93,13 +93,15 @@
     "@microsoft/loader-load-themed-styles": "2.0.17",
```

Comment:
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=30313&commitId=28d9ea51ce8de10ae71c6df692d7d4be41b50a67&env=prod&clientType=FLUENTUI)


  <details><summary><b>Pivot - Overflow</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Pivot - Overflow.Root.Narrow - Last tab selected.chromium.png|0| Removed|
|Pivot - Overflow.Root.Narrow - Overflow menu.chromium.png|0| Removed|

</details>


  <details><summary><b>react-charting-PieChart</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-PieChart.Basic - Dark Mode.default.chromium.png|6674| Changed|
|react-charting-PieChart.Basic.default.chromium.png|5462| Changed|
|react-charting-PieChart.Basic - RTL.default.chromium.png|5460| Changed|

</details>


  <details><summary><b>react-charting-SankeyChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-SankeyChart.Basic.default.chromium.png|317| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>